.travis.yml

```diff
@@ -1,10 +1,13 @@
+sudo: required
```

Comment:
This is what Travis recommends for using Trusty: https://docs.travis-ci.com/user/trusty-ci-environment/